code/src/com/sixsq/nuvla/server/resources/deployment_set/utils.clj

```diff
@@ -97,6 +99,7 @@
 (def transition-operational-status {::tk/on action-operational-status ::tk/guards [sm/guard-can-manage?]})
 (def transition-recompute-fleet {::tk/on action-recompute-fleet ::tk/guards [sm/guard-can-edit?
                                                                              guard-fleet-filter-defined?]})
+(def transition-auto-update {::tk/on action-auto-update ::tk/guards [sm/guard-can-manage?]})
```

Comment:
The action is always available even for deployment group without auto-refresh flag enabled. Maybe this make sense to keep it. Or to allow only admin to call it from job-engine. Something to discuss

Reply:
Indeed, it would make sense to not allow the action for a deployment group without auto-refresh flag enabled - will try to do that.
Restrict to admin only would also make sense I guess - is the call being done from the job-engine an admin call or do I need to change something there too ?